### PR TITLE
Lifestyle section

### PR DESCRIPTION
**Pull request by jalfonze**

### CODE REVIEW

**Comment by jankdc** [comment]
Deploy preview processing.

Built with commit 142c3a709bfdb9dbccb6df53505ff7508ec3b8ae

https://app.netlify.com/sites/elegant-meitner-4ff78c/deploys/5a2247f30b79b71e61ccda29

**Comment by jalfonze** [comment]
i just wanna see how this one looks

**Comment by jankdc** [comment]
here's how it looks https://5a2247f30b79b71e61ccda29--elegant-meitner-4ff78c.netlify.com/

File: _posts/2017-08-19-why-travel.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Why Travel"
+date: 2016-02-19
+categories:
+  - Lifestyle
+description: 
+image: https://lensdump.com/i/dMILD
```
Comment (jankdc) [inline _posts/2017-08-19-why-travel.markdown]: this should be: https://i.lensdump.com/i/dMILD.jpg

File: _posts/2017-08-19-why-travel.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+image-sm: https://lensdump.com/i/dMILD
```
Comment (jankdc) [inline _posts/2017-08-19-why-travel.markdown]: same here: https://i.lensdump.com/i/dMILD.jpg

**Comment by jankdc** [comment]
let me know when you're happy with this so I can merge it to the live site

**Comment by jalfonze** [comment]
I'm happy with this. Merge.

**Pull request merged**